### PR TITLE
added protected route

### PR DESCRIPTION
**Pull request by prabhathc**

### CODE REVIEW

File: client/src/components/AuthProvider.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  const value = {
+    token,
+    handleLogin,
+    handleLogout,
+  };
+
+  return <AuthContext.Provider value={value}>{children}</AuthContext.Provider>;
+}
+export { useAuth };
```
Comment (prabhathc) [inline client/src/components/AuthProvider.jsx:41]: Do I need this?

Comment (KenanOzbelli) [reply client/src/components/AuthProvider.jsx:41]: It shows the child components thru the protected route.

**Review by KenanOzbelli — COMMENTED** [review]
LGTM

**Review by KenanOzbelli — APPROVED** [review]
LGTM Again

**Pull request merged**